### ISSUE TITLE
Dragon piece visual bug with black and white piece set on Firefox

### ISSUE THREAD

**Issue by travv0** [WandererXII/lishogi#829]
**Exact URL where the bug happened**
https://lishogi.org/training/zJNPX

**Steps to reproduce the bug**
Use black and white piece set on Firefox and look at the above puzzle, or any other place you can see a dragon piece.

**What did you expect to happen?**
The dragon piece looks like it's supposed to.

**What happened instead?**
It looks like this:
![image](https://github.com/WandererXII/lishogi/assets/3605522/41a00827-2bc0-47bb-aedf-dee587873b9e)

**Operating system and browser version**
Windows 10, Firefox 126.0

**Comment by gbtami** [comment]
Reneded OK in Chrome. Maybe this is a Firefox SVG rendering issue? Idk.
Ping @visualdenniss

**Comment by visualdenniss** [comment]
Strange issue indeed. Renders OK in both Chrome and Safari for me. But Firefox seems to experience a rendering issue with this SVG. I only provided the SVGs, which I created in Photoshop and I don't see any issues in the original files.
Perhaps something to try to fix would be:
Instead of embedding the SVG directly into the HTML using a data URI (if it was implemented this way), perhaps referencing an external SVG file could solve. Otherwise, no idea about possible fixes. Seems Firefox specific issue.
